test/grub2_widgets_test.rb

```diff
@@ -617,8 +617,8 @@ def stub_widget_value(id, value)
     expect(bootloader.sections.default).to eq "openSUSE"
   end
 
-  it "lists all available sections" do
-    expect(subject.list).to eq([
+  it "enlists all available sections" do
```

Comment:
I found the absence of `describe "#list"` a little bit strange, but it's consistent with the rest of this rspec, so...